src/tbp/monty/frameworks/models/motor_policies.py

```diff
@@ -498,14 +546,16 @@ def fixme_undo_last_action(self):
         else:
             raise ValueError(f"Invalid action: {last_action}")
 
-    def post_action(self, action: Action) -> None:
+    def post_action(
+        self, action: Action, state: Optional[MotorSystemState] = None
+    ) -> None:
         self.action = action
         self.timestep += 1
         self.episode_step += 1
-        state_copy = self.convert_motor_state()
+        state_copy = self.convert_motor_state(state)
         self.action_sequence.append([action, state_copy])
 
-    def convert_motor_state(self):
+    def convert_motor_state(self, state: MotorSystemState):
```

Comment:
I'd have to open this in PyCharm to be sure, but I think it would tell me that this could be a static method, since `self` is no longer referenced.

Similar to my comments above, I think this method belongs on the MotorSystemState and not the policy.